src/Api/bikeApi.ts

```diff
@@ -1,40 +1,35 @@
 import {BASE_URL} from "./urls"
-import {axiosHandleResponse} from "./ApiUtils"
+import {axiosHandleResponse, getRequestConfig} from "./ApiUtils"
 import axios from "axios"
 
 const bikes_url = BASE_URL + "bikes/";
 
-export enum BikeState {
-    Working, InService, Blocked,
+export enum BikeStatus {
+    Available, Rented, Reserved, Blocked,
 }
 
 export interface Bike {
     id: string;
-    status: BikeState;
+    status: BikeStatus;
     station?: {
         id: string;
         name: string;
     };
 }
 
 export const postBike = async (station: string) => {
-    let res =  await axios({
-        method: 'post',
-        url: bikes_url,
-        data: {
-            stationId: station,
-        }
-    });
+    axios.post(bikes_url, { stationId: station }, getRequestConfig());
 }
 
 export const getBikes = async () => {
-    return axios.get(bikes_url).then(r =>  axiosHandleResponse(r));
+    return axios.get(bikes_url, getRequestConfig())
```

Comment:
W sumie nadal w żadnej funkcji nie mamy obsługi błędów. To nie twoje zadanie, ale przydałoby się to pododawać